dataload/import_to_elasticsearch.py

```diff
@@ -165,6 +166,10 @@ def import_to_elasticsearch(files, clean):
 
     time.sleep(1)
 
+    with open(os.path.join(current_dir, 'charity_names.json')) as fd:
```

Comment:
The actual json file is in the repo, so it should always work.  The script is just for when it needs updating